src/entries/watasalim.ts

```diff
@@ -9,4 +10,5 @@ export default {
   topic: AcademicTopics.Next,
   imagePath: "/images/watasalim.png",
   githubUrl: "https://github.com/narze/awesome-salim-quotes",
+  ButtonLabel: ButtonLabel.Seemore,
```

Comment:
Make sure you apply all fixes to all files. **Please run locally** to see if your build success and also run format.

Reply:
Sorry for disturbing krub.Next time will run locally before submitting.